internal/oci/container.go

```diff
@@ -346,6 +346,23 @@ func (c *Container) exitFilePath() string {
 	return filepath.Join(c.dir, "exit")
 }
 
+// IsRunning is a function that checks if a container's init pid exists.
+// It is used to check a container state when we don't want a `$runtime state` call
+func (c *Container) IsRunning() bool {
```

Comment:
I am thinking, maybe change this to `IsAlive` or `IsInitAlive`, or something like this, as `IsRunning` is a bit misleading

Reply:
or `IsInitExists` as the documentation says